backend/src/main/java/io/debezium/configserver/service/postgres/PostgresConnectorIntegrator.java

```diff
@@ -138,7 +137,7 @@ public FilterValidationResult validateFilters(Map<String, String> properties) {
 
         PostgresConnectorConfig config = new PostgresConnectorConfig(Configuration.from(properties));
 
-        try (PostgresConnection connection = new PostgresConnection(config.getJdbcConfig())) {
+        try (PostgresConnection connection = new PostgresConnection(config.getJdbcConfig(), PostgresConnection.CONNECTION_GENERAL)) {
```

Comment:
This should be handled by @ani-sha for [DBZ-5317](https://issues.redhat.com/browse/DBZ-5317) not here.

Reply:
That's fine, I can rebase against `main` after her work is addressed.